ui-tests/src/test/resources/features/integrations/api-provider.feature

```diff
@@ -1049,9 +1145,9 @@ Feature: API Provider Integration
       | parameters.id | body.id |
     And sleep for jenkins delay or "2" seconds
     And click on the "Done" button
-    And publish API Provider integration
+    And click on the "Save" link
+    And publish integration
     And navigate to the "Integrations" page
     Then wait until integration "TODO Integration base path" gets into "Running" state
     When select the "TODO Integration base path" integration
-    Then verify the displayed API Provider URL matches regex ^https://i-todo-integration-base-path-syndesis.*/api$
-
+    Then verify the displayed API Provider URL matches regex ^https://i-todo-integration-base-path-syndesis.*$
```

Comment:
I believe the `/api$` portion was the point of the test - is it no longer shown? If so, that would be a regression

Reply:
I can change it back if it is meant to behave that way, It made sense to me to have it without the `/api` as you can define more paths.

Reply:
The point is that the base path is specified in the swagger file https://github.com/syndesisio/syndesis-qe/blob/master/ui-tests/src/test/resources/swagger/connectors/todo.json#L13 and this should be shown in UI. See https://github.com/syndesisio/syndesis/issues/4471

Reply:
Aha, okay my fault, I checked the URL when creating the API provider from scratch.